.github/workflows/format-code.yml

```diff
@@ -16,9 +16,10 @@ jobs:
     runs-on: macos-11
     steps:
       - uses: actions/checkout@v3
+      ## Update internal list of formulae to the latest
+      - run: brew update
       - name: Install Clang-Format
         run: brew install clang-format
-      - run: clang-format --version
```

Comment:
This is redundant. The install log above already displays the version.

Reply:
True. But I already merged it, sorry 😬

Reply:
Oh wait I actually didn't, weird. So you can still change the PR.

Reply:
No worries, I added it 😁